### PR TITLE
[AKS] Add network plugin none support

### PR DESCRIPTION
**Pull request by mattstam**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

File: src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py [hunk 1]
```diff
@@ -3438,4 +3438,27 @@ def test_aks_create_with_crg_id(self, resource_group, resource_group_location):
...
+            'location': resource_group_location,
+            'resource_type': 'Microsoft.ContainerService/ManagedClusters',
+            'ssh_key_value': self.generate_ssh_keys(),
+        })
+
+        create_cmd = 'aks create --resource-group={resource_group} --name={name} --network-plugin=none ' \
+                     '--location={location} --ssh-key-value={ssh_key_value} -o json'
+        
+        self.cmd(create_cmd, checks=[
+            self.check('provisioningState', 'Succeeded'),
```
Comment (FumingZhang) [inline src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py:3463]: I suppose we'd better also check whether the network plugin is "none". No need to update the recording file though.

**Comment by FumingZhang** [comment]
Some of the CIs failed due to lack of the recording file for the newly added test case. I have queued a [pipeline](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=52001513&view=results) to generate one. May download it from [here](https://dev.azure.com/msazure/_apis/resources/Containers/87491260/live%20test%20reports?itemPath=live%20test%20reports%2Fext-recordings%2Ftest_aks_create_with_network_plugin_none.yaml) and add this to the `test/latest/recordings` folder.

File: src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py (hunk 1 above)
Comment (mattstam) [reply src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py:3463]: Thanks Fuming, I have added that check.

**Comment by mattstam** [comment]
> Some of the CIs failed due to lack of the recording file for the newly added test case. I have queued a [pipeline](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=52001513&view=results) to generate one. May download it from [here](https://dev.azure.com/msazure/_apis/resources/Containers/87491260/live%20test%20reports?itemPath=live%20test%20reports%2Fext-recordings%2Ftest_aks_create_with_network_plugin_none.yaml) and add this to the `test/latest/recordings` folder.

Thank you!

**Review by FumingZhang — APPROVED** [review]
The PR looks good to me, but need to resolve the conflicts about release version, since host group id took 0.5.54.

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**